GP05MOAS-GL363/GP05MOAS-GL363_D00006_ingest.csv

```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-01-FLORDM000,telemetered,Available,A 3-wavelength fluorometer (FLORTM) was deployed in place of a 2-wavelength (FLORDM). This is the parser for 3-wavelength fluorometer
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GP05MOAS-GL363/D00006/merged-from-glider/gp_*.mrg,GP05MOAS-GL363-04-CTDGVM000,telemetered,Available,
```

Comment:
looks good; confirmed via deploy.csv that for the FLORTM in place of FLORDM the refdes is correct (stays as GP05MOAS-GL363-01-FLORDM000)
Same comment as above re filename_mask.